webpack.config.js

```diff
@@ -0,0 +1,22 @@
+const path = require('path');
+
+module.exports = {
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+    path: path.resolve(__dirname, 'dist'),
+  },
```

Comment:
- [ ] Kindly add devServer configuration to your webpack.config.js., just below output config In this format: 
```
devServer: {
    contentBase: path.join(process.cwd(), 'dist'),
    compress: true,
  },
```